browser/net/brave_proxying_web_socket.cc

```diff
@@ -179,6 +179,10 @@ void BraveProxyingWebSocket::ContinueToHeadersReceived() {
   OnHeadersReceivedComplete(net::OK);
 }
 
+void BraveProxyingWebSocket::OnFailure(const std::string& message,
+                                       int32_t net_error,
+                                       int32_t response_code) {}
```

Comment:
should this be a noop @iefremov ?

Reply:
I guess so, it is a noop in the chrome counterpart